app/konnect/analytics/dashboard.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Analytics dashboard
+---
+
+![service graph](/assets/images/products/konnect/analytics/konnect-summary-dashboard.png){:.image-border}
+
+
+You can access a summary dashboard by navigating to [**Analytics** > **Summary**](https://cloud.konghq.com/analytics). The summary dashboard shows performance and health statistics of all your APIs across your organization on a single page and provides greater insights into your service usage.
+
+From {% konnect_icon analytics %} Analytics, you can view dashboards and access historical data for a range greater than 30 days. You can view the Analytics summary dashboard to track traffic, errors by error code, and latency across all services in your organization.
+
+These categories measure trends by comparing metrics across fixed comparable time intervals and plotting the data points. For example, hour-over-hour, day-over-day, week-over-week, and month-over-month.
+You can view a graph for the **Requests**, **Error Rate**, and **P99 Latency** categories:
+
+* **Requests**: This graph displays the total number of HTTP requests categorized by successful and failed requests over the specified time interval.
```

Comment:
We should change this by:
* Remove the 3 bullet points.
* Change the text to something like "This number displays the total number of HTTP requests over the specified time interval. This number also includes any requests made to incorrect routes.